src/amp/components/Analytics.tsx

```diff
@@ -6,7 +6,7 @@ export interface AnalyticsModel {
 	title: string;
 	fbPixelaccount: string;
 	comscoreID: string;
-	section?: string;
+	section: string;
```

Comment:
`section` is used in `Analytics` (src/amp/components/Analytics.tsx) and `Ad` (/src/amp/components/Ad.tsx). 

It would make sense for it to be always defined, but currently it is sometimes defined as `''`.